### DISCUSSION TITLE
Windows 10 LTSC for CHIA

### DISCUSSION THREAD

**Discussion by Tezuya69** [Chia-Network/chia-blockchain#6057] [Support]
Hello,
do is good the Windows 10 LTSC for chia ?
THX :)

**Comment by tiny0little** [comment]
if Ubuntu 20.04 is too difficult to use, this windows will work with no problem

**Reply by Tezuya69** [reply]
Thank you ... i'm novice ... and i don't want problems

**Comment by BrandtH22** [comment]
@Tezuya69 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
